.github/workflows/build-jsp.yml

```diff
@@ -54,7 +54,7 @@ jobs:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
     - name: Package JSPs
       run: >
-        mvn -s .github/workflows/mvn-settings.xml -B -pl UnicodeJsps install package
+        mvn -s .github/workflows/mvn-settings.xml -B -pl UnicodeJsps install package -DCLDR_DIR=${GITHUB_WORKSPACE}/cldr
```

Comment:
Tests must be run here. Good catch.

Reply:
Tests were run here before, but across the several commits sometimes I needed the CLDR path and other times I didn't. :shrug: